etc/cp-ksql-server/values.yaml

```diff
@@ -0,0 +1,115 @@
+  # resources, such as Minikube. If you do want to specify resources, uncomment the following
+  # lines, adjust them as necessary, and remove the curly braces after 'resources:'.
+  # limits:
+  #  cpu: 100m
+  #  memory: 128Mi
+  # requests:
+  #  cpu: 100m
+  #  memory: 128Mi
+
+## Custom pod annotations
+podAnnotations: {}
+
+## Node labels for pod assignment
+## Ref: https://kubernetes.io/docs/concepts/configuration/assign-pod-node/
+nodeSelector: {}
+
+## Taints to tolerate on node assignment:
+## Ref: https://kubernetes.io/docs/concepts/configuration/taint-and-toleration/
+tolerations: []
+
+## Pod scheduling constraints
+## Ref: https://kubernetes.io/docs/concepts/scheduling-eviction/assign-pod-node/#affinity-and-anti-affinity
+affinity: {}
+
+## Monitoring
+## JMX Settings
+## ref: https://docs.confluent.io/current/ksql/docs/operations.html
+jmx:
+  port: 5555
+
+## Prometheus Exporter Configuration
+## ref: https://prometheus.io/docs/instrumenting/exporters/
+prometheus:
+  ## JMX Exporter Configuration
+  ## ref: https://github.com/prometheus/jmx_exporter
+  jmx:
+    enabled: true
+    image: solsson/kafka-prometheus-jmx-exporter@sha256
+    imageTag: 6f82e2b0464f50da8104acd7363fb9b995001ddff77d248379f8788e78946143
+    imagePullPolicy: IfNotPresent
+    port: 5556
+
+    ## Resources configuration for the JMX exporter container.
+    ## See the `resources` documentation above for details.
+    resources: {}
+
+## External Access
+## ref: https://kubernetes.io/docs/concepts/services-networking/service/#type-loadbalancer
+external:
+  enabled: false
+  type: LoadBalancer
+  externalTrafficPolicy: Cluster
+  port: 8088
+
+## Headless mode
+## ref: https://docs.confluent.io/current/ksql/docs/installation/server-config/index.html
+ksql:
+  headless: true
+  queries: ""
+  # For example:
+  # queries: |
+  #   CREATE STREAM pageviews_original (viewtime bigint, userid varchar, pageid varchar)
+  #       WITH (kafka_topic='pageviews', value_format='DELIMITED');
+
+## You can list load balanced service endpoint, or list of all brokers (which is hard in K8s).  e.g.:
+## bootstrapServers: "PLAINTEXT://dozing-prawn-kafka-headless:9092"
+kafka:
+  bootstrapServers: ""
+
+## e.g. http://gnoble-panther-cp-schema-registry:8081
+cp-schema-registry:
+  url: ""
+
+# KSQL configuration options
+## ref: https://docs.confluent.io/current/ksql/docs/installation/server-config/config-reference.html
+configurationOverrides:
+    "ksql.logging.processing.topic.auto.create": true
+    "ksql.logging.processing.stream.auto.create": true
+    "ksql.service.id": radar_default_
+    "ksql.persistence.default.format.key": AVRO
+    "ksql.persistence.default.format.value": AVRO
+    "ksql.sink.replicas": 3
+    "ksql.streams.replication.factor": 3
+    "ksql.internal.topic.replicas": 3
+    "ssl.truststore.location": "/etc/kafka/secrets/kafka.client.truststore.jks"
+    "ssl.truststore.password": confluent
```

Comment:
I think these are not required in the default stack (as we use PLAINTEXT), but should not cause any issues with the deployment (just mentioned as might be confusing to users if they are not aware of it)

Reply:
Thanks, removed.